eras/babbage/impl/src/Cardano/Ledger/Babbage/PParams.hs

```diff
@@ -350,32 +351,37 @@ babbagePParamsPairs ::
   PParamsHKD Identity era ->
   [a]
 babbagePParamsPairs pp =
-  uncurry (.=) <$> babbagePParamsHKDPairs (Proxy @Identity) pp
+  uncurry (.=)
+    <$> babbagePParamsHKDPairs (Proxy @Identity) pp
+      ++ [ "decentralization" .= Aeson.Null
+         , "extraPraosEntropy" .= Aeson.Null
+         , "minUTxOValue" .= Aeson.Null
+         ]
 
 instance FromJSON (BabbagePParams Identity era) where
   parseJSON =
     withObject "PParams" $ \obj ->
       BabbagePParams
-        <$> obj .: "minFeeA"
-        <*> obj .: "minFeeB"
+        <$> obj .: "txFeePerByte"
+        <*> obj .: "txFeeFixed"
         <*> obj .: "maxBlockBodySize"
         <*> obj .: "maxTxSize"
         <*> obj .: "maxBlockHeaderSize"
-        <*> obj .: "keyDeposit"
-        <*> obj .: "poolDeposit"
-        <*> obj .: "eMax"
-        <*> obj .: "nOpt"
-        <*> obj .: "a0"
-        <*> obj .: "rho"
-        <*> obj .: "tau"
+        <*> obj .: "stakeAddressDeposit"
+        <*> obj .: "stakePoolDeposit"
+        <*> obj .: "poolRetireMaxEpoch"
+        <*> obj .: "stakePoolTargetNum"
+        <*> parseAsRational (obj .: "poolPledgeInfluence")
+        <*> parseAsRational (obj .: "monetaryExpansion")
+        <*> parseAsRational (obj .: "treasuryCut")
```

Comment:
This is unnecessary, since FromJSON instance will do the same thing
```suggestion
        <*> obj .: "poolPledgeInfluence"
        <*> obj .: "monetaryExpansion"
        <*> obj .: "treasuryCut"
```